src/Models/Account.js

```diff
@@ -253,14 +253,14 @@ class Account {
      * @param {Object} data - Contains {
      *  email: String,
```

Comment:
`email` type should be lowercase. It's in the same docs as the changes in this file for the `brandingId` so I think we should fix this as well while we're here.